web/modules/custom/dpl_identity/dpl_identity.routing.yml

```diff
@@ -0,0 +1,7 @@
+dpl_identity.color:
+  path: "/admin/config/dpl_identity"
+  defaults:
+    _form: '\Drupal\dpl_identity\Form\IdentityConfigForm'
+    _title: "Identity Color"
```

Comment:
Consider whether this form should be solely focused on colors or whether we expect it to expand into other areas in the future.

If so then color should probably be removed from much of all this configuration.

It is notably not part of the `IdentityConfigForm` class name.